app/code/core/Mage/Adminhtml/Block/Customer/Edit/Tab/Wishlist/Grid/Renderer/Description.php

```diff
@@ -23,6 +23,7 @@ class Mage_Adminhtml_Block_Customer_Edit_Tab_Wishlist_Grid_Renderer_Description
 {
     public function render(Varien_Object $row)
     {
-        return nl2br(htmlspecialchars($row->getData($this->getColumn()->getIndex())));
+        $value = $row->getData($this->getColumn()->getIndex());
+        return is_string($value) ? nl2br(htmlspecialchars($value)) : '';
```

Comment:
>  It can't be an array without changing the related code.
 
It's possible for 3rd-party to change the related code. If that's the case, throwing a fatal error is the correct behavior:

```suggestion
        return $value !== null ? nl2br(htmlspecialchars($value)) : '';
```

PHP is moving towards strict typing (PHP9), which is essentially throwing fatal errors on wrong types; instead of implicitly type casting and not throw anything as in pre-PHP7. Throwing fatal errors in strict type is supposed to help debugging and catch the bugs earlier, and help us write dependable code. For null deprecation, we should focus on fixing the `null` part and leaves it to PHP to enforce the typing rule. If we start to do type checking, we would need to throw fatal errors (because that's what strict typing means.)

The problem we have with OpenMage is that `null` is the default value with magic getter. It didn't have any problem and was cool then, but now, sentiments have changed. The [opinion](https://www.techtarget.com/whatis/definition/strongly-typed) is that "weakly typed programming languages can produce erroneous or unpredictable results".